winui/Kanban/overview.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: About WinUI Kanban control | Syncfusion
+description: Learn here all about introduction of Syncfusion Essential Studio WinUI Kanban (SfKanban) control, its features and more.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+
+# Overview of WinUI Kanban (SfKanban)
+
+The Syncfusion WinUI Kanban control is an efficient way to visualize a workflow at each stage of completion. It facilitates clear planning and provides a transparent view of work progression. SfKanban offers a variety of features that are essential for monitoring tasks throughout the software development cycle.
+
+## Key features
+
+* Work-in-Progress (WIP) Limit: Allows users to set task limits for each stage, preventing overload and improving efficiency.
```

Comment:
**Work-in-progress (WIP) limit**: Users can set task limits per stage, preventing overload and improving efficiency.
* **Cards**: Display tasks, progress, and priority.
* **Columns**: Organize work into columns for stages like to-do, in-progress, and completed.
* **Swim lanes**: Organize tasks based on projects, teams, or custom criteria.
* **Interaction**: Includes card tapping, column expansion/collapse, and drag-and-drop.
* **Events** : Triggered when cards are dragged, repositioned, or tapped.
* **Appearance customization**: Customize each element using templates, e.g., column headers, cards, and swim lane headers.
* **Localization and RTL**: Easily adapt the control to any supported language or culture. Render the board right-to-left (RTL) to suit RTL languages.
* **Themes**: Built-in light and dark themes automatically adjust to system settings.